utils/misc.py

```diff
@@ -0,0 +1,38 @@
+from bs4 import BeautifulSoup
+from tempfile import TemporaryFile
+from the_mines.download.get_html import download_default_html
+
+
+def umlaut(word_with_umlaut):
+    """Insert accented chars where applicable
+
+    Args:
+        word_with_umlaut(str): a word containing an umlaut
+    Returns:
+        word with actual umlaut
+    """
+    if "\\xc3\\xb6" in word_with_umlaut:
+        word_with_umlaut = word_with_umlaut.replace("\\xc3\\xb6", "ö")
+    if "\\xc3\\xbc" in word_with_umlaut:
+        word_with_umlaut = word_with_umlaut.replace("\\xc3\\xbc", "ü")
+    if "\\" in word_with_umlaut:
+        word_with_umlaut = word_with_umlaut.replace("\\", "")
+    return word_with_umlaut
+
+
+def get_default_matchday(matchday_or_season):
+    raw_html = download_default_html()
+    with TemporaryFile("w+") as tmp:
+        tmp.write(raw_html)
+        tmp.seek(0)
+        soup = BeautifulSoup(tmp, "html.parser")
+
+        title = soup.title.text
+        if matchday_or_season == "matchday":
+            ind = title.index(".")
+            matchday = title[ind - 2 : ind]
+        elif matchday_or_season == "season":
+            ind = title.index("/")
+            matchday = title[ind - 4 : ind + 5]
+
+    return matchday.strip()
```

Comment:
I think this section would benefit from some comments.  Especially since we are using indices.  Is it possible to rewrite this section using something list decomposition? Something like:
```python
# just an example, I don't know what the ouput you are working with looks like
title = "Bayern gegen Dortmund (10.10.2010, DFB-Pokal)"
team1, vs, team2, date, competition = title.split(" ")
```
That way we _know_ what each component from the string is even if we don't use it.  It is much more understandable than using an index.

Also I think this method would benefit from an early return pattern:
```python
if condition:
    return this
else condition:
    return that
```

Reply:
Very sexy suggestions. I will look into these.

Reply:
Tried this, but it would require 3 different .split() calls. One for each state of a match (not played, played, and live) because the position of where `season` or `matchday` are in the list. What do you think?

Reply:
I actually change my mind on this comment.  I found the way I was doing it was unreliable because teams _sometimes_ have multiple words in their name (i.e. `union berlin`).  I would suggest using regex groups 👍 Thanks for testing this idea out!